### DISCUSSION TITLE
Social Auth not following proxy settings in config file

### DISCUSSION THREAD

**Discussion by georgedavid29** [netbox-community/netbox-docker#1171] [Getting Help]
### Current Behavior

module doesnt follow the proxy listed in the config file.
error below: 

<class 'social_core.exceptions.AuthFailed'>

Authentication failed: HTTPSConnectionPool(host='login.microsoftonline.com', port=443): Max retries exceeded with url: /common/oauth2/token (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3>: Failed to establish a new connection: [Errno 101] Network is unreachable'))

Python version: 3.9.18
NetBox version: 3.7.3
Plugins: None installed

I'm confident the proxy settings are working because the netbox news populates properly when I log in with the local accounts. also the proxy logs show the allowed traffic. i don't see any denied traffic in the proxy logs for login.microsoftonline.com in the logs for the netbox ip.

### Expected Behavior

successful login into the netbox app via azure sso 

### Docker Compose Version

n/a

### Docker Version

```text
n/a
```

### The git Revision

n/a

### The git Status

```text
n/a
```

### Startup Command

n/a

### NetBox Logs

```text
n/a
```

### Content of docker-compose.override.yml

```yaml
n/a
```

**Comment by Jding2159** [comment]
Anyone have an update for this or is able to point me in the correct direction? Running into the same issue. The Netbox container is sitting behind an nginx proxy to serve TLS.

Python Version: 3.11.6
Netbox Version: 4.0.7

extra.py
```
LOGIN_REQUIRED=True
REMOTE_AUTH_ENABLED=True
REMOTE_AUTH_AUTO_CREATE_USER=True
REMOTE_AUTH_BACKEND='social_core.backends.azuread.AzureADOAuth2'
SOCIAL_AUTH_AZUREAD_OAUTH2_KEY='yzx'
SOCIAL_AUTH_AZUREAD_OAUTH2_SECRET='xyz'
SOCIAL_AUTH_REDIRECT_IS_HTTPS=True
```

error 
```
Authentication failed: HTTPSConnectionPool(host='login.microsoftonline.com', port=443): Max retries exceeded with url: /common/oauth2/token (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f3>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
```

tried
- Exporting the cert path as environmental variable

**Reply by Jding2159** [reply]
Hi, anyone have any idea how to resolve the above?

**Reply by Domoninic** [reply]
I think you may need to set HTTP_PROXIES for outbound connections.

Ref : https://netboxlabs.com/docs/netbox/en/stable/configuration/system/#http_proxies
